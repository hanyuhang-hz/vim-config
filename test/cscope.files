./hyh/Test.c
./hyh/hyh.cpp
./hyh/hyh.hpp
./hyh/hyh.java
./test.c
./test1.c
./xxxx.c
