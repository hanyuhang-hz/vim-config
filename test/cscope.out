cscope 15 $HOME/github/vim-config-install/test -q 0000000022 0000000667
	@hyh/Test.c

1 
	g‹¡


	@hyh/hyh.cpp

2 
	~"hyh.hµ
"

3 
	~<io¡»am
>

5 
usšg
 
Çme¥aû
 
	g¡d
;

7 
	$g‘
(
a
) {

8 
cout
 << "‹¡" <<
’dl
;

10 
	}
}

12 þas 
	cGass
 {

13 
	mpublic
:

14 
g‘
();

15 
	m´iv©e
:

16 
m
;

	@hyh/hyh.hpp

3 
g‘
(
a
);

	@hyh/hyh.java

1 
	g‹¡


	@test.c

1 
	~"ioùl.h
"

2 
	~"hyh.hµ
"

4 
	$‹¡1
() {

5 
	`‹¡_1
();

6 
	`g‘
();

7 
‹¡


8 
	}
}

10 
	$‹¡2
() {

11 
	`g‘
();

12 
	}
}

14 
	$hª
() {

16 
	}
}

18 
	$hyh3333
() {

20 
	}
}

22 
	$maš
() {

25 
	}
}

	@test1.c

2 
	$‹¡_1
() {

4 
	}
}

	@xxxx.c

1 
	$xxxx
() {

3 
	}
}

	@
1
.
1
/usr/include
7
70
hyh/Test.c
hyh/hyh.cpp
hyh/hyh.hpp
hyh/hyh.java
test.c
test1.c
xxxx.c
